MainTel/MainTel-Troubleshooting/MTCS-Troubleshooting/TMA-troubleshooting/TMA-Troubleshooting-OSS-Fails-to-Turn-On-index.rst

```diff
@@ -0,0 +1,181 @@
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+
+
+
+.. _Title-of-Troubleshooting-Procedure-Critical-Step-1:
+
+ 1. Go to Level 1 Machinery Room and bring a Philips screwdriver with you. The Machinery Room is in the left door in the picture below.
+
+
+
+
+.. figure:: ./_static/level-1.png
+    :name: level 1
+
+
+
+
+2. The screens in the front panel of one or both of the chiller cabinets would display an alarm. On the top, the red band may say: **“STATUS: NOT ACKNOWLEDGED”** or **“STATUS: ALARM”**
+
+
+
+
+.. figure:: ./_static/chiller_2.jpeg
+    :name: chiller 2
+
+
+
+
+.. figure:: ./_static/alarm_menu.jpeg  
+    :name: alarm menu
+
+
+
+
+
+The large grey box on the cabinet's side (purple arrow) shows a blinking red light and an interlock in the screen.
+
+
+
+.. figure:: ./_static/grey_box_cabinets.jpeg
+    :name: grey box cabinets
+
+    .. figure:: ./_static/interlock_screens.jpeg
+        :name: interlock screens
+
+3. There are two reset buttons inside the chiller cabinet that needs to be pressed. From the side of the cabinet, use the screwdriver to remove two screws that are located on top. This will allow you to remove the cover panel.
+ Each cabinet has two panels. In chiller #2 the reset buttons are within the furthermost panel (Image below), while in chiller#1 they are found under the panel closer to the front face of the chiller.
+
+.. figure:: ./_static/furthermost_panel.jpeg
+    :name: remove panel
+
+    .. figure:: ./_static/Hand.jpeg
+        :name: 
+
+
+4. Press each of the red reset buttons inside the cabinet for 3 seconds (TBC: order does not matter):
+
+.. figure:: ./_static/reset_button_1.jpeg
+ :name: buttons
+
+
+
+.. figure:: ./_static/reset_button_2.jpeg
+ :name: reset
+
+
+
+
+   .. _Title-of-Troubleshooting-Procedure-Final-Step:
+
+
+
+5. Press the reset button on the large grey box on the side of the cabinet. The red alarm light should stop blinking.
+
+6. Acknowledge the alarms by pressing the tick mark on the bottom of the screen. After perhaps a minute or two, the front screen should show **"STATUS:OK"**
+
+.. figure:: ./_static/acknowledge_alarm.jpeg
+    :name:
+
+7. Repeat **steps 2-6** with the other chiller, if it shows an alarm.    
+
+8. Back to the control room, reset the alarm in the TMA EUI **"Safety System"** screen. Click on the red **"OSS malfunction"** and press **"RESET SELECTED"**. The red colour should dissapear.
```

Comment:
SAFETY SYSTEMS is written in all caps in the EUI.

Reply:
I don't understand this comment.